From 9ef0cc445e682bc4858c2933fb6712385e18d862 Mon Sep 17 00:00:00 2001
From: lyz <lyz@rock-chips.com>
Date: Sat, 10 Jan 2015 19:08:16 +0800
Subject: [PATCH] usb: manual delay 8s to enable connect pc

Android generate Serial number too later than set 1 to
sys/class/android_usb/android0/enable. So here we have to manual
delay some time to enable usb connect in usb controler driver,
otherwise adb read a wrong serial number via usb then CTS test
will fail.

Signed-off-by: lyz <lyz@rock-chips.com>
---
 drivers/usb/dwc_otg_310/dwc_otg_pcd_linux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/dwc_otg_310/dwc_otg_pcd_linux.c b/drivers/usb/dwc_otg_310/dwc_otg_pcd_linux.c
index 3ee983b9b18c..d22768ce21ee 100755
--- a/drivers/usb/dwc_otg_310/dwc_otg_pcd_linux.c
+++ b/drivers/usb/dwc_otg_310/dwc_otg_pcd_linux.c
@@ -1748,7 +1748,7 @@ static void dwc_otg_pcd_work_init(dwc_otg_pcd_t *pcd,
 		/* host mode,enter usb phy mode */
 		pldata->dwc_otg_uart_mode(pldata, PHY_USB_MODE);
 	}
-	schedule_delayed_work(&pcd->check_id_work, HZ);
+	schedule_delayed_work(&pcd->check_id_work, 8 * HZ);
 }
 
 #endif /* DWC_HOST_ONLY */
-- 
2.35.3

